src/cript/api/api.py

```diff
@@ -85,18 +86,44 @@ class API:
     _db_schema: dict = {}
     _http_headers: dict = {}
 
-    def __init__(self, host: Union[str, None], token: [str, None]):
+    def __init__(self, host: Union[str, None] = "", token: [str, None] = "", config_file_path: Union[Path, str] = ""):
```

Comment:
I was debating between `None` or `""` and I thought if I put it as `None` then the user can do `cript.API()` without specifying anything, but that could also work as well because then the user can create a CRIPT API easier.

I can change it to `None` for now, and we can change it later if needed